docs/deploycstor.md

```diff
@@ -134,6 +135,33 @@ cStor can be provisioned in your Kubernetes cluster by performing the following
    ---
    ```
 
+In auto pool method, you don't have to select the disks which is detected by ndm for cStor Pool creation. You can create a file called *openebs-config.yaml* in your master node and add the following contents into the file.  Goto Step 5 to apply the modified *openebs-config.yaml* for cStor pool creation.
```

Comment:
In the auto pool....you must not select the disks which are detected by NDM for creating cStor pools.
Goto = Go to...for creating cStor pools.